From 2fb188349e802771873fd42eafb012a420c0fd23 Mon Sep 17 00:00:00 2001
From: Kevin Mark <kmark937@gmail.com>
Date: Tue, 2 Jul 2019 14:27:16 -0400
Subject: [PATCH 18/49] many ports hit this warning:
 ../sysdeps/unix/sysv/linux/check_native.c:46: warning: implicit declaration
 of function 'memset' ../sysdeps/unix/sysv/linux/check_native.c:46: warning:
 implicit declaration of function 'memset'

snipped from suse
---
 sysdeps/unix/sysv/linux/check_native.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/sysdeps/unix/sysv/linux/check_native.c b/sysdeps/unix/sysv/linux/check_native.c
index 6e6624abe1..b577f4ff12 100644
--- a/sysdeps/unix/sysv/linux/check_native.c
+++ b/sysdeps/unix/sysv/linux/check_native.c
@@ -23,6 +23,7 @@
 #include <stddef.h>
 #include <stdint.h>
 #include <stdlib.h>
+#include <string.h>
 #include <time.h>
 #include <unistd.h>
 #include <net/if.h>
-- 
2.22.0

